_posts/languages/php/2000-01-01-start.md

```diff
@@ -172,6 +172,7 @@ These extensions will be installed dynamically if required in the `composer.json
     PHP 7.4 and above)
 * Lua: [`lua`](https://www.php.net/manual/en/book.lua.php)
 * Gettext: [`gettext`](https://www.php.net/manual/en/book.gettext.php)
+* mcrypt: [`mcrypt`](https://pecl.php.net/package/mcrypt)
```

Comment:
Shouldn't we explain that ≤ 7.2 it's bundled with PHP, and as a module for  ≥ 7.3?

Reply:
Yes maybe, you're right.